operator/apis/platform/v1alpha1/securedcluster_types.go

```diff
@@ -291,9 +293,12 @@ type CollectorContainerSpec struct {
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,order=2
 	ImageFlavor *CollectorImageFlavor `json:"imageFlavor,omitempty"`
 
-	// Set this to 'true' to prevent translation of the collection method, e.g. to enforce using EBPF as an exception.
+	// When the "collection" field is set to the deprecated value "EBPF", then
+	// setting this to "true" prevents conversion of the collection method to
+	// "CORE_BPF". This could be used in exceptional situations when "EBPF" has
+	// to be enforces.
```

Comment:
```suggestion
	// to be used.
```

Reply:
The form `enforces` seems like an error. I guess this was supposed to be `enforced`, but IMHO `used` would be better. Up to you.